container-storage-setup.sh

```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/dss-fifo1
-PIPE2=/run/dss-fifo2
+PIPE1=/run/css-fifo1
+PIPE2=/run/css-fifo2
```

Comment:
This will create issues with multiple css instances running, right? We probably should introduce some locking so that only one instance of container-storage-setup is running at a time.